internal/common/ingest/testfixtures/event.go

```diff
@@ -8,6 +8,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 
+	"github.com/armadaproject/armada/internal/armada/configuration"
```

Comment:
We may want to add a `_test` suffix to the file name to avoid these being built into the final binary.